services/types.d.ts

```diff
@@ -0,0 +1,40 @@
+export type MessageResponse = {
```

Comment:
i think u should move this to types, cases might happen were u might reuser this types outside of services